code/datums/antagonists/cult.dm

```diff
@@ -12,6 +12,10 @@
 	return ..()
 
 /datum/antagonist/cult/proc/add_objectives()
+	if(!SSticker.mode.cult_team)
```

Comment:
No, don't make it dependant on the mode. Initial assignment should create and set team along with objectives not the antag. Late conversions should use converter team. Also this has nothing to do with objectives so don't bundle it together. If you're really lazy make it static on the datum.